be/src/olap/aggregate_func.h

```diff
@@ -580,6 +581,53 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_BITMAP_UNION, OLAP_FIELD_TYPE_
         : public AggregateFuncTraits<OLAP_FIELD_AGGREGATION_BITMAP_UNION, OLAP_FIELD_TYPE_OBJECT> {
 };
 
+template <>
+struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_QUANTILE_UNION, OLAP_FIELD_TYPE_QUANTILE_STATE> {
+    static void init(RowCursorCell* dst, const char* src, bool src_null, MemPool* mem_pool,
+                     ObjectPool* agg_pool) {
+        DCHECK_EQ(src_null, false);
+        dst->set_not_null();
+
+        auto* src_slice = reinterpret_cast<const Slice*>(src);
+        auto* dst_slice = reinterpret_cast<Slice*>(dst->mutable_cell_ptr());
+
+        // we use zero size represent this slice is a agg object
+        dst_slice->size = 0;
+        auto* quantile_state = new QuantileState<double>(*src_slice);
+
+        mem_pool->mem_tracker()->Consume(quantile_state->get_serialized_size());
```

Comment:
Why not using ```sizeof(QuantileState)``` instead of ```quantile_state->get_serialized_size()```?
Which one is more precise?

Reply:
The result of sizeof() is not incorrect, which is much lower than actual memory usage, cause we have vector or obj ptr in the member variables.
The result of  get_serialized_size is closer to real memory usage.